internal/cephfs/util.go

```diff
@@ -34,7 +34,10 @@ import (
 type volumeID string
 
 func execCommandErr(ctx context.Context, program string, args ...string) error {
-	_, _, err := util.ExecCommand(ctx, program, args...)
+	_, stdErr, err := util.ExecCommand(ctx, program, args...)
```

Comment:
@Yuggupta27 using go-ceph binding for purgeVolume should fix the issue can you please check